app/src/main/AndroidManifest.xml

```diff
@@ -39,6 +39,11 @@
                 <action android:name="org.shadowice.flocke.andotp.intent.ENTER_DETAILS" />
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
+            <intent-filter>
```

Comment:
I believe this will launch andOTP any time an NFC tag is read, is this the functionality we'd want? We might want to register a separate `BroadcastReceiver` in `AuthenticateActivity` and have the read be registered there, so that the user has to open andOTP and then read the tag vs the tag read event always opening andOTP (since the NFC tag reading may be unrelated).